src/core/shape_inference/include/roi_align_shape_inference.hpp

```diff
@@ -73,5 +73,68 @@ void shape_infer(const ov::op::v3::ROIAlign* op, const std::vector<T>& input_sha
 }
 
 }  // namespace v3
+namespace v9 {
+
+template <class T>
+void shape_infer(const ov::op::v9::ROIAlign* op, const std::vector<T>& input_shapes, std::vector<T>& output_shapes) {
```

Comment:
The body of this `v9::shape_infer` function for `v9::ROIAlign` looks like an exact copy of the `v3::shape_infer` above.
Can we reuse this piece of code as a common function to don't repeat it twice?